contracts/BridgeCommittee.sol

```diff
@@ -96,29 +116,36 @@ contract BridgeCommittee {
         } else if (messageType == BRIDGE_OWNERSHIP) {
             address newOwner = decodeBridgeOwnershipMessage(message);
             _transferBridgeOwnership(newOwner);
+        // Need to check the message type is legit (e.g. can't be 10000)
         } else {
             // if the message type is not for the committee, submit it to the bridge
             bridge.submitMessage(message);
         }
-        emit MessageProcessed(message);
+        emit MessageProcessedEvent(message);
     }
 
+    // Is this the regular way to upgrade?
```

Comment:
short answer is no. Typically upgrades are handled by a single admin signer or a multisig (that handles most of the details) I'm unclear how upgrading via another contract should work so this function is subject to change. (working on this now)